packages/gatsby-kontent-components/README.md

```diff
@@ -52,6 +52,16 @@ import { RichTextElement } from "@kentico/gatsby-kontent-components"
       resolveLinkedItem={linkedItem => {
         return <pre>{JSON.stringify(linkedItem, undefined, 2)}</pre>
       }}
+      resolveDomNode={(domNode, domToReact) => {
```

Comment:
Love this API and the overall usage of resolvings for RichText ❤

Reply:
THX!